docs/how-to-guides/curating-the-editor-experience.md

```diff
@@ -14,10 +14,25 @@ Users have the ability to lock and unlock blocks via the editor. The locking UI
 
 Keep in mind that each block you want to lock will need to be individually locked as desired. There is not a way to mass lock blocks currently. 
 
-**Lock patterns or templates**
+**Apply block locking to patterns or templates**
 
 When building patterns or templates, theme authors can use these same UI tools to set the default locked state of blocks. For example, a theme author could lock various pieces of a header. Keep in mind that by default, users with editing access can unlock these blocks. [Here’s an example of a pattern](https://gist.github.com/annezazu/acee30f8b6e8995e1b1a52796e6ef805) with various blocks locked in different ways and here’s more context on [creating a template with locked blocks](https://make.wordpress.org/core/2022/02/09/core-editor-improvement-curated-experiences-with-locking-apis-theme-json/). You can build these patterns in the editor itself, including adding locking options, before following the [documentation to register them](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-patterns/).
 
+**Apply content locking in patterns or templates**
```

Comment:
"Content locking" is actually the opposite of what's actually happening. We're locking down the structural/design attributes of the block, while keeping just the content editable. I think this will be quite confusing in the current wording.

Reply:
Agreed. This is exactly why I wanted your read of it :D Do you think we should roll with calling this "content only editing"?

Reply:
Yea, I think that resonates (from feedback/responses of my article at least).